frontend/public/components/cluster-settings/openid-idp-form.tsx

```diff
@@ -26,10 +26,10 @@ export class AddOpenIDPage extends PromiseComponent {
     name: 'openid',
     mappingMethod: 'claim',
     clientID: '',
-    clientSecretFileContent: '',
-    claimPreferredUsernames: [],
-    claimNames: [],
-    claimEmails: [],
+    clientSecret: '',
```

Comment:
similar to htpasswd formatting, do you wanna update that as well?

Reply:
HTPasswd does actually use a file. Client secret for OpenID is just a string.